x-pack/platform/plugins/shared/streams_app/public/components/data_management/schema_editor/schema_changes_review_modal.tsx

```diff
@@ -139,14 +293,16 @@ export function SchemaChangesReviewModal({
           {i18n.translate(
             'xpack.streams.schemaEditor.confirmChangesModal.fieldsWillBeUpdatedText',
             {
-              defaultMessage: 'The fields below will be updated.',
+              defaultMessage:
+                'Some fields below will be updated. Any error from validation will also appear here',
```

Comment:
```suggestion
                'Some fields below will be updated. Any error from validation will also appear here.',
```
Missing a period here. Would it be accurate to say, "The following fields will be updated. Validation errors will appear here."

Reply:
@mdbirnstiehl Thanks for catching the missing period.

We had to replace the `The following` with `Some` fields because this message appears on top of a table which displays a bunch of fields, not all would be updated. Let me know if this is fine